### PR TITLE
cli: append warning when --template option is ignored

### LINKED ISSUE #390: cli: --template is silently ignored when --format template is not passed
`--template` option is silently ignored when `--format <table|json>` or no `--format` is passed.

### PR DESCRIPTION
**Pull request by tnir**
to avoid `--template` is silently ignored when `--format <table|json>` or no `--format` is passed.

Closes #390 #394

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

### CODE REVIEW

File: internal/standalone/config/config.go [hunk 1]
```diff
@@ -97,6 +97,13 @@ func New(c *cli.Context) (Config, error) {
 }
 
 func (c *Config) Init() (err error) {
+	if c.Template != "" {
```
Comment (knqyf263) [inline internal/standalone/config/config.go]: Don't we have to display a warning message in the following case? What do you think?
```
$ trivy --format template  golang:1.12-alpine    
```

In short, it is like this condition.
```
if c.Format == "template" && c.Template == ""{
}
```

Comment (tnir) [reply internal/standalone/config/config.go]: @knqyf263 Thanks for your review. Added the test case for it in a8bebfd and implemented it in a8be49e.

Comment (knqyf263) [reply internal/standalone/config/config.go]: Thanks! I know it is your fault, but I think it is better to add this check for consistency.

**Comment by knqyf263** [comment]
Thanks a lot!

**Pull request merged**